tutorial/lesson_11_cross_compilation.cpp

```diff
@@ -45,7 +45,7 @@ int main(int argc, char **argv) {
     // program on.  For example, if you compile and run this file on
     // 64-bit linux on an x86 cpu with sse4.1, then the generated code
     // will be suitable for 64-bit linux on x86 with sse4.1.
-    brighter.compile_to_file("lesson_11_host", args);
+    brighter.compile_to_file("lesson_11_host", args, "lesson_11_host");
```

Comment:
I think in this file it would be better to use a uniform function name for all variants, because that's what you would do when cross-compiling